### PR TITLE
Enable prettier for links file

### PR DESCRIPTION
**Pull request by ijlee2**
## Description

this enables prettier for the links file so that we can follow up and change some links in another PR

### CODE REVIEW

**Review by mansona — CHANGES_REQUESTED** [review]
I know you say to review this commit by commit but can we please keep the prettier and the actual link changes in different PRs to make sure that we don't miss things. 

For example, this PR adds a regression where the `/release/` part of the Guides link is being removed 👍

**Comment by mansona** [comment]
also I'm surprised by the "remove the copy in the website" because this should never have been added to the website in the first place 🤔 

I can't find the PR where it was added, do you have the link?

File: addon/constants/links.js [hunk 1]
```diff
@@ -1,128 +1,167 @@
...
+      {
+        href: 'https://api.emberjs.com',
+        name: 'API Reference',
+        type: 'link',
+      },
+      {
+        href: 'https://cli.emberjs.com',
+        name: 'CLI Guides',
+        type: 'link',
+      },
```
Comment (ijlee2) [inline addon/constants/links.js:20]: `https://guides.emberjs.com` is redirected to `https://guides.emberjs.com/release` and matches the pattern for API and CLI Guides below. The API and CLI links are redirected to the corresponding release pages. I don't think it's good to have a mix of patterns.

**Review by mansona — APPROVED** [review]

**Comment by locks** [comment]
We discussed this PR at the meeting today and we are merging the formatting changes so we can proceed with updating the actual links :)
I updated the title to better reflect the content of the PR.

**Pull request merged**